guides/doc-Upgrading_and_Updating/master.adoc

```diff
@@ -96,6 +96,9 @@ endif::[]
 // Reclaiming Database Space
 include::common/modules/proc_reclaiming-postgresql-space.adoc[leveloffset=+3]
 
+// Updating Templates, Parameters, Lookup Keys and Values
+include::topics/post_upgrade_template_checks.adoc[leveloffset=+3]
+
```

Comment:
We need to find out why this module was removed and therefore whether it's okay to put it back.